datasets/covid19_google_mobility/pipelines/mobility_report/pipeline.yaml

```diff
@@ -35,100 +35,88 @@ dag:
     - operator: "KubernetesPodOperator"
       description: "Run CSV transform within kubernetes pod"
       args:
-        task_id: "mobility_report_transform_csv"
+        task_id: "mobility_report"
         startup_timeout_seconds: 600
         name: "mobility_report"
         namespace: "composer"
         service_account_name: "datasets"
         image_pull_policy: "Always"
         image: "{{ var.json.covid19_google_mobility.container_registry.run_csv_transform_kub }}"
         env_vars:
-          SOURCE_URL: "https://www.gstatic.com/covid19/mobility/Global_Mobility_Report.csv"
-          SOURCE_FILE: "files/data.csv"
-          TARGET_FILE: "files/data_output.csv"
+          PIPELINE_NAME: "{{ var.json.covid19_google_mobility.mobility_report.pipeline_name }}"
+          SOURCE_URL: "{{ var.json.covid19_google_mobility.mobility_report.source_url }}"
+          SOURCE_FILE: "{{ var.json.covid19_google_mobility.mobility_report.source_file }}"
+          TARGET_FILE: "{{ var.json.covid19_google_mobility.mobility_report.target_file }}"
+          CHUNKSIZE: "{{ var.json.covid19_google_mobility.mobility_report.chunksize }}"
+          PROJECT_ID: "{{ var.value.gcp_project }}"
+          DATASET_ID: "{{ var.json.covid19_google_mobility.mobility_report.dataset_id }}"
+          TABLE_ID: "{{ var.json.covid19_google_mobility.mobility_report.table_id }}"
```

Comment:
Let's parameterize only `PROJECT_ID`. Everything else we can make transparent to readers of our code (open source).

Reply:
Done